src/components/templates/Camp/components/MapRoomOverlay.tsx

```diff
@@ -97,12 +97,7 @@ export const MapRoomOverlay: React.FC<MapRoomOverlayProps> = ({
     (e) => {
       e.stopPropagation();
 
-      if (isExternalUrl(room.url)) {
-        openRoomUrl(room.url);
-      } else {
-        window.location.href = getRoomUrl(room.url);
-      }
-
+      openRoomUrl(room.url);
       enterCampRoom(room);
```

Comment:
These changes are insufficient to handle all room entering in this component. See https://github.com/sparkletown/sparkle/pull/888